registry/handlers/mail.go

```diff
@@ -0,0 +1,45 @@
+package handlers
+
+import (
+	"errors"
+	"net/smtp"
+	"strings"
+)
+
+// Mail provides fields of email configuration for sending.
+type Mail struct {
```

Comment:
This struct is not used outside this package and we don't intend to support its usage outside of this package.

This also applies to `LogHook`. If we choose to use these elsewhere, we'll make that change at that time.